hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -179,7 +181,7 @@ public void init() throws DBException {
     String table = getProperties().getProperty(TABLENAME_PROPERTY, TABLENAME_PROPERTY_DEFAULT);
     try {
```

Comment:
If you merge the try above into this try you would have something like:

    if( THREAD_COUNT.getAndIncrement() == 0) {
        String table = getProperties().getProperty(TABLENAME_PROPERTY, TABLENAME_PROPERTY_DEFAULT);
        try {
            if (connection == null) {
                // Initialize if not set up already.
                connection = ConnectionFactory.createConnection(config);

                // Validate the table name.
                final TableName tName = TableName.valueOf(table);
                connection.getTable(tName).getTableDescriptor();
            }
        } catch (java.io.IOException e) {
            throw new DBException(e);
       }
    }

Reply:
Good point, merged em.